README.md

```diff
@@ -60,6 +60,29 @@ Documentation on the `v2` structure can be found [here](https://github.com/ONSdi
 e.g."http://localhost:8000/quick-launch?schema_url=http://localhost:7777/1_0001.json&version=v1"
 ```
 
+### Commands for Formatting & Linting
+Install dependencies using `npm install` for formatting and linting
```

Comment:
I don't think `npm install` is enough on its own, we also need to set the Node version somewhere